spread.yaml

```diff
@@ -22,6 +22,18 @@ prepare: |
     echo "Spread is running as $(id)"
     [ "$REUSE_PROJECT" != 1 ] || exit 0
     ./spread-tests/spread-prepare.sh
+    # trusty support is under development right now
+    # we special-case the release until we have officially landed
+    if ["$(lsb_release -sc)" == "trusty"]
+    then
+        add-apt-repository ppa:thomas-voss/trusty
+        apt update && apt upgrade
+        apt install systemd snapd
```

Comment:
systemd should be a dependency of snapd but I'm okay if that's just a temporary thing.